api/testdata/complete-scenarios/test6.xml

```diff
@@ -977,13 +977,13 @@
               <Investigations>
                 <Investigation ID="1">
                   <Agency>Unknown</Agency>
+                  <InvestigationDate>
```

Comment:
These golden file changes look benign, but just to be sure, the 3 SF86 test scenarios that changed should be submitted to e-QIP test and a note on the PR should be added that that was done.